ps_emailalerts.php

```diff
@@ -571,6 +571,7 @@ public function hookActionUpdateQuantity($params)
         $check_oos = ($product_has_attributes && $id_product_attribute) || (!$product_has_attributes && !$id_product_attribute);
 
         if ($check_oos &&
+        Validate::isLoadedObject($product) &&
```

Comment:
```suggestion
            Validate::isLoadedObject($product) &&
```

Reply:
Isn't it something you also have to do for `hasAttributes`?
Should we simply add a return if the product object is not correctly loaded?

Reply:
@PierreRambaud I did it and CS fixer complained :D

Reply:
Regarding the question - Maybe we can just do this check right after `$product = new Product($id_product, false, $id_lang, $id_shop, $context);` and return immediately.

Reply:
@Hlavtox we don't want to do anything for non-existing product, your idea to have an early return is ok :)

Reply:
That's the idea yeah! :+1:

Reply:
@PierreRambaud @kpodemski Updated